roles/mistral/defaults/main.yml

```diff
@@ -1,4 +1,4 @@
-mistral_version: 0.13
+mistral_version: 1.3.2-99
```

Comment:
I guess you mean something like what's used here: https://github.com/johandahlberg/ansible-st2/blob/use_deb_packages/roles/st2/tasks/main.yml? If so I can fix it now, shouldn't be to tricky.